src/System.Private.ServiceModel/tools/scripts/CleanUpHttpsPort.cmd

```diff
@@ -0,0 +1,32 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+echo Remove Https configuration
+
+net session > NUL
+if NOT [%_ErrorLevel%]==[0] (
```

Comment:
I forgot to set _ErrorLevel.  I'll set __ExitCode directly to avoid two statements.